.gitignore

```diff
@@ -64,4 +64,7 @@ documentation/rebuildpdf.bat
 .build_source/
 # direnv
 .envrc
-.env
\ No newline at end of file
+.env
+
+# CI generated files
+*effective-pom*
```

Comment:
Why generate effective poms in production?

Reply:
For the effective pom, I was thinking it wouldn't cost much to leave them for the moment to debug if we have an issue, but I can remove them if you prefer.

Reply:
removed